lib/index.js

```diff
@@ -154,6 +155,20 @@ class WebpackPluginServe extends EventEmitter {
     invalid.tap(key, (filePath) => this.emit('invalid', filePath, compiler));
     watchClose.tap(key, () => this.emit('close', compiler));
 
+    if (this.options.waitForBuild) {
+      // track the first build of the bundle
+      this.state.compiling = new Promise((resolve) => {
+        this.once('done', () => resolve());
```

Comment:
Can be shortened to ```this.once('done', resolve);```

Reply:
it could :) in certain situations I like to be a bit more verbose.